### DISCUSSION TITLE
How to use MapLibre

### DISCUSSION THREAD

**Discussion by itishermann** [rnmapbox/maps#1389] [Q&A] [answered]
Hello everyone
I'm trying to use maplibre with react-native-mapbox-gl on android, but i don't seem to see any documentation on this. 
I've successfully installed it as described in the documentation but it still asks for an access token from Mapbox.
Can someone give me a tutorial or some help please ?

**Comment by mfazekas** [comment, marked as answer]
I assumed you installed using:
https://github.com/react-native-mapbox-gl/maps/blob/master/android/install.md#using-maplibre

What do you mean `still asks for access token from Mapbox`?! If you connect to mapbox servers for map access token is still required.

**Reply by itishermann** [reply]
Yes i've installed it using the docs and i've used the example provided here https://github.com/react-native-mapbox-gl/maps/blob/master/docs/GettingStarted.md#show-a-map but i commented the access token line.
There is the error i'm getting
` ERROR  Mapbox error {com.agendav2}[Setup]: loading style failed: HTTP status code 401 {"level": "error", "message": "{com.agendav2}[Setup]: loading style failed: HTTP status code 401", "tag": "Mbgl"}`

**Reply by mfazekas** [reply]
Yes by default the style url loads from mapbox servers, and that requires `accessToken`. So you need to provide you own styleURL. I'm not sure if we have example for that.

https://github.com/react-native-mapbox-gl/maps/blob/master/docs/MapView.md

This docs describes how get an url for MapTiler:
https://docs.maptiler.com/maplibre-gl-native-ios/ios-uikit-basic-get-started/
